dex/src/main/scala/com/wavesplatform/dex/market/OrderBookActor.scala

```diff
@@ -269,7 +262,7 @@ object OrderBookActor {
   )
 
   object MarketStatus {
-    implicit val fmt: Writes[MarketStatus] = { ms =>
+    implicit val marketStatusWrites: OWrites[MarketStatus] = { ms =>
```

Comment:
Because `Writes` returns a `JsValue`, not a `JsObject`. Sometimes we need to receive an object. In this branch we need it in `OrderBookHttpInfo.getMarketStatus`. `SimpleResponse` due to implementation can't accept `JsValue`